osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -102,6 +102,7 @@ private void load(OsuConfigManager config, NotificationOverlay notificationOverl
         {
             showHud = config.GetBindable<bool>(OsuSetting.ShowInterface);
             showHud.ValueChanged += hudVisibility => visibilityContainer.FadeTo(hudVisibility ? 1 : 0, duration);
+            showHud.ValueChanged += v => Progress.AllowSeeking = v;
```

Comment:
This seems wrong when `AllowSeeking` was false and hud made visible.

Reply:
Also there is no reason to separate to two event additions.

Reply:
yea thats right its only allowed to change when a replay / autoplay is loaded

Reply:
ok so i didnt find any issue making the visibility container not always present.
fixes the problem as well.

Reply:
since the updates the elements in the `visibilityContainer` receive are always from sources which are not triggered through Positional or NonPositional input.

Reply:
git-blame revealed #3762 was the issue. Are you sure it is not regressed?

Reply:
The Hold to quit button is in a different `FillFlowContainer` and is not Hidden by shift+tab

Reply:
None of the elements in the `visibilityContainer` need to  receive input at any time except the `Progress` which is only supposed to accept it when visible.

Reply:
Actually, `ModDisplay` -> `ModIcon` receives positional input to display a tooltip but it works fine with this change. Confirmed.